charts/storage-apps/values.yaml

```diff
@@ -73,7 +73,7 @@ minio:
   # -- Chart
   chart: "minio"
   # -- [minio Helm chart](https://github.com/minio/minio/tree/master/helm-releases) version
-  targetRevision: "5.0.13"
+  targetRevision: "5.4.0"
```

Comment:
Are there any additional changes (to the chart) that should be surfaced in the changelog?

Reply:
Added the missing releases tags, so we have all the info there

Reply:
A quick summary of the contents would have been very much in order.